.github/workflows/deploy.js.yml

```diff
@@ -2,7 +2,7 @@ name: Deploy
 
 on:
   push:
-    branches: [ next ]
+    branches: [ next, main ]
```

Comment:
I'm not sure why the branch naming was changed, but this should cause the gh-pages integration I setup to automatically deploy on merge.